### PR TITLE
Geometry labels

### PR DESCRIPTION
**Pull request by Sybrand**
Create a separate table for each layer, that contains only the centroid of the source layer. This allows for a separate layer that only contains labels, not entire geometries and thus avoiding duplication of labels that may occur when serving up geometries that span multiple tiles.

### CODE REVIEW

File: fetch_feature_layer.py [hunk 1]
```diff
@@ -30,6 +33,28 @@ def get_column_type(value):
     raise Exception(f'unknown type {type(value)}')
 
 
+def create_point_table_schema(meta_data: MetaData, data: dict, table_name: str, srid: int) -> Table:
+    """
+    """
```
Comment (andrea-williams) [inline fetch_feature_layer.py]: Ah yes. The cheater's way of adding code "documentation" so linting won't complain.

**Review by andrea-williams — APPROVED** [review]
No obvious concerns from me, but I'm out of the loop on what's going on with the tileserver...

**Review by micheal-w-wells — COMMENTED** [review]
I'm also just catching on this, but interested to see how it works.  I've only tried to sync data from DataBC and not ESRI, and also not in python.

More a general question/comment than feedback relevant to the PR, but here I am anyway:  Is there a search feature being passed to ESRI or are we grabbing the whole object list for the whole layer in fetch_object_list ?  If it works it works it's not like a human is doing all those http calls to grab each object, but I'm curious if it's possible to search by a feature or if it's planned.

I can imagine the object list for some has to be insane (insane like the features with so many duplicate vertices that help create these issues to begin with).

File: fetch_feature_layer.py (hunk 1 above)
Comment (Sybrand) [reply fetch_feature_layer.py]: That method was dumb anyway. I've removed it.

**Comment by Sybrand** [comment]
> I'm also just catching on this, but interested to see how it works. I've only tried to sync data from DataBC and not ESRI, and also not in python.
> 
> More a general question/comment than feedback relevant to the PR, but here I am anyway: Is there a search feature being passed to ESRI or are we grabbing the whole object list for the whole layer in fetch_object_list ? If it works it works it's not like a human is doing all those http calls to grab each object, but I'm curious if it's possible to search by a feature or if it's planned.
> 

Right now it's grabbing the entire layer. For our use case at least, that's the desired functionality.
I don't have any intention to implement any kind of search right now - but if it's something someone needs there's no reason it can't be added. Right now I'm only focusing on our teams needs, but I'd love to make this repo be a general solution that multiple teams can use.

> I can imagine the object list for some has to be insane (insane like the features with so many duplicate vertices that help create these issues to begin with).

The "duplication" issue I mentioned relates to how vector tiles are served up and rendered. A geometry may span multiple tiles, so it ends up with a label for each tile. For the layers I'm pulling right now, the data is nice and clean.

**Review by conbrad — APPROVED** [review]

**Review by JulianForeman — APPROVED** [review]

**Pull request merged**